test/hotspot/jtreg/gc/shenandoah/oom/TestAllocOutOfMemory.java

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
Or is this new test subsuming several old tests?

Reply:
Correct, we rolled the `TestAlloc` tests into one file.